### DISCUSSION TITLE
想询问一下参考文献里的DOI怎么去掉

### DISCUSSION THREAD

**Discussion by Gilles1551** [sjtug/SJTUThesis#823] [提问] [answered]
如题，想要的格式是
> [9] 江向东 . 互联网环境下的信息处理与图书管理系统解决方案 [J/OL]. 情报学
> 报,1999,18(2);4[2000-01-18].http://www.chinainfo.gov.cn/periodical/gbxb/gbxb99/gbxb9902
> 03. 

但是编译出来的格式是

> [9] 江向东 . 互联网环境下的信息处理与图书管理系统解决方案 [J/OL]. 情报学
> 报,1999,18(2);4[2000-01-18]. DOI: http://www.chinainfo.gov.cn/periodical/gbxb/gbxb99/gbxb9902
> 03. 

想请教一下这个“DOI”怎么去除

**Comment by Gilles1551** [comment]
> @article{huling2021,
   title={头戴式眼动追踪系统的视线估计方法研究},
   author={胡玲},
   tertiaryauthor={宋宗玺;王锋},
   publisher={中国科学院大学(中国科学院西安光学精密机械研究所)},
   typeofwork={硕士},
   year={2021},
   keywords={头戴式;眼动追踪;眼动特征提取;三维视线估计模型},
   abstract={视线估计技术是一种新兴智能人机视觉交互方式,是通过构建眼部图像中眼动特征和注视点之间的模型,来解算人眼注视方向或注视点位置的技术。本文在分析头戴式系统集成化结构特性下,重点研究两部分内容,包括近红外光照明下瞳孔特征提取算法和建立基于此特征的视线估计模型。本文主要工作和研究成果如下},
   doi={10.27605/d.cnki.gkxgs.2021.000060},
   databaseprovider={cnki},
}

bibtex格式如上，主要是从知网上引用的文章，这个doi部分会额外显示一个“DOI”的字样

**Comment by AlexaraWu** [comment, marked as answer]
`biblatex` 添加选项 `doi=false`。

**Reply by Gilles1551** [reply]
添加了这个之后把链接整个都去掉了，图书馆只要求去掉“DOI：”的前缀，要保留后面的链接

**Reply by AlexaraWu** [reply]
链接放在 url 域里。
```bibtex
@article{huling2021,
  title            = {头戴式眼动追踪系统的视线估计方法研究},
  author           = {胡玲},
  tertiaryauthor   = {宋宗玺;王锋},
  publisher        = {中国科学院大学(中国科学院西安光学精密机械研究所)},
  typeofwork       = {硕士},
  year             = {2021},
  keywords         = {头戴式;眼动追踪;眼动特征提取;三维视线估计模型},
  abstract         = {视线估计技术是一种新兴智能人机视觉交互方式,是通过构建眼部图像中眼动特征和注视点之间的模型,来解算人眼注视方向或注视点位置的技术。本文在分析头戴式系统集成化结构特性下,重点研究两部分内容,包括近红外光照明下瞳孔特征提取算法和建立基于此特征的视线估计模型。本文主要工作和研究成果如下},
  doi              = {10.27605/d.cnki.gkxgs.2021.000060},
  url              = {http://www.chinainfo.gov.cn/periodical/gbxb/gbxb99/gbxb9902},
  databaseprovider = {cnki}
}
```
